packages/subcommands/src/lib/SubcommandMappings.ts

```diff
@@ -71,11 +72,10 @@ export interface ChatInputSubcommandMappingValue {
 	to: ChatInputSubcommandToProperty | string;
 
 	/**
-	 * Should this command be ran if no input is given
-	 *
+	 * Select whether you want to execute a command class method or a command registered in the store.
 	 * @since 3.0.0
 	 */
-	default?: boolean;
+	type: SubCommandType;
```

Comment:
I feel like it'd be better to have this as optional and default it to using `method` in the handlers code like the current version of the plugin does

Reply:
Done